test/fulfillmentSpec.js

```diff
@@ -348,6 +349,12 @@ describe('PUT /fulfillment', function () {
     })
 
   it('should not double spend when transfer is executed multiple times', function * () {
+    // Pre-fund hold account to allow double spending of source transfer
+    // This could happen if, for ex, 2 txn are escrowed, then first is executed twice.
```

Comment:
It seems like we should fix the first transaction being executed twice, then we wouldn't need this hack?